pyproject.toml

```diff
@@ -81,6 +81,9 @@ doc = [
     "nbconvert",
     "mike >=1.0.0"
 ]
+codecs = [
```

Comment:
**Comment**
FYI, the now standard way to define dependency groups has been codified in [PEP-735](https://peps.python.org/pep-0735/). It replaces `[project.optional-dependencies]`.